internal/app/export/service.go

```diff
@@ -22,22 +21,22 @@ type Service interface {
 var _ Service = (*exporter)(nil)
 
 type exporter struct {
-	Nc     *nats.Conn
-	Mqtt   mqtt.Client
-	Logger log.Logger
-	Cfg    config.Config
-	Routes []routes.Route
+	NatsConn *nats.Conn
+	Mqtt     mqtt.Client
+	Logger   log.Logger
+	Cfg      config.Config
+	Routes   []routes.RouteIF
```

Comment:
I have noted this before: please do not invent your standard on naming interfaces.